examples/site_selection_example.jl

```diff
@@ -0,0 +1,18 @@
+using ADRIA
+using ADRIA: site_selection
```

Comment:
Here, you are importing `site_selection` explicitly, so it makes sense that calling the function directly works, but I don't see `site_selection_scens` being similarly imported, nor do I see it being exported by the ADRIA package. 

How does it work without an "undefined" error?